src/store/modules/ADempiere/dictionary/window/actions.js

```diff
@@ -180,7 +180,7 @@ export default {
 
             const storedTab = rootGetters.getStoredTab(windowUuid, tabAssociatedUuid)
             const { table_name } = storedTab
-            dispatch('runReport', {
+            dispatch('startReport', {
               parentUuid: tabUuid,
               containerUuid: process.uuid,
```

Comment:
Validate here only `FindReport`
Use this UUID: `a42bad0c-fb40-11e8-a479-7a0060f0aa01`

Reply:
Also you can use this for report value instead: `FinReport`